### PR TITLE
Fix fuzzing versions in serialization tests

### PR DESCRIPTION
**Pull request by pmorie**
@liggitt @deads2k @pweil-

### CODE REVIEW

File: pkg/api/serialization_test.go [hunk 1]
```diff
@@ -389,10 +392,17 @@ func TestSpecificKind(t *testing.T) {
 		return
 	}
 	seed := int64(2703387474910584091) //rand.Int63()
-	fuzzInternalObject(t, "", item, seed)
-	roundTrip(t, osapi.Codec, item)
-	roundTrip(t, v1beta3.Codec, item)
-	roundTrip(t, v1.Codec, item)
+	for i := 0; i < fuzzIters; i++ {
+		t.Logf("About to test %v with \"\"", kind)
```
Comment (liggitt) [inline pkg/api/serialization_test.go]: backticks let you avoid escaping quotes

**Comment by liggitt** [comment]
is `version` used in `FuzzerFor`? what is this fixing?

**Comment by pmorie** [comment]
@liggitt It's passed in `TestTypes`, I wanted them to be uniform since it doesn't help much to have a test to debug a specific type if it doesn't do exactly what the 'real' test does.

**Comment by pmorie** [comment]
[test]

**Comment by liggitt** [comment]
LGTM

**Comment by ironcladlou** [comment]
[merge]

**Pull request merged**